lib/Notes/NotesAccordion/components/NotesList/NotesList.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  FormattedDate,
+} from 'react-intl';
+import memoize from 'memoize-one';
+
+import { MultiColumnList } from '@folio/stripes-components';
+import {
+  noteShape,
+  columnShape,
+} from '../../NotesAccordion';
+
+export default class NotesList extends React.Component {
+  static propTypes = {
+    columnsConfig: PropTypes.arrayOf(columnShape),
+    notes: PropTypes.arrayOf(noteShape).isRequired,
+    onNoteClick: PropTypes.func.isRequired,
+  }
+
+  getItems() {
+    return this.props.notes
+      .map(note => {
+        const {
+          id,
+          title,
+          lastSavedDate,
+          lastSavedUserFullName,
+        } = note;
+
+        return {
+          id,
+          date: (
+            <FormattedDate
+              value={lastSavedDate}
+              year="numeric"
+              month="numeric"
+              day="numeric"
+            />
+          ),
+          updatedBy: lastSavedUserFullName,
+          title,
+        };
+      });
+  }
+
+  getColumnNames = memoize((config) => {
```

Comment:
Do not see the purpose to use memoize here as the number of columns can not be too high